src/bp-media/classes/class-bp-rest-media-albums-endpoint.php

```diff
@@ -905,6 +905,8 @@ public function prepare_item_for_response( $album, $request ) {
 			'title'         => $album->title,
 			'privacy'       => $album->privacy,
 			'media'         => $album->media,
+			'group_name'    => ( isset( $album->group_name ) ? $album->group_name : '' ),
+			'visibility'    => ( isset( $album->visibility ) ? $album->visibility : '' ),
 			'user_email'    => $album->user_email,
```

Comment:
Just a question : why this field "user_email" is added in the response. The email can be private and shouldn't be exposed so easily...